etna/packages/etna-js/dispatchers/__tests__/work-dispatcher.spec.jsx

```diff
@@ -57,40 +56,12 @@ describe('workDispatcher', () => {
     const action = {
       type: WORK,
       work_type: 'upload',
-      otherArgs
+      command: otherArgs
```

Comment:
Small change to work dispatcher's calling convention, making the command a sub item in the structure.  Makes destructuring simpler.